wren-ai-service/src/eval/ask/__main__.py

```diff
@@ -361,6 +379,9 @@ def eval(prediction_results_file: Path, dataset_name: str, ground_truths: list[d
     if not Path("./outputs").exists():
         Path("./outputs").mkdir()
 
+    with open(f"./src/eval/data/{DATASET_NAME}_optimal_ddl.json", "r") as f:
```

Comment:
you should only read this file only if USE_OPTIMAL_SCHEMA is True

Reply:
Thanks for the suggestion, the code is updated.